packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -87,6 +92,48 @@ const capturePhaseEvents = new Set([
   TOP_WAITING,
 ]);
 
+const isArray = Array.isArray;
+
+function dispatchEventsForPlugins(
+  topLevelType: DOMTopLevelEventType,
+  eventSystemFlags: EventSystemFlags,
+  nativeEvent: AnyNativeEvent,
+  targetInst: null | Fiber,
+  rootContainer: Element | Document,
+): void {
+  const nativeEventTarget = getEventTarget(nativeEvent);
+  const syntheticEvents = [];
+
+  for (let i = 0; i < plugins.length; i++) {
+    const possiblePlugin: PluginModule<AnyNativeEvent> = plugins[i];
+    if (possiblePlugin) {
```

Comment:
*I* prefer not coercing these values, only because I've been burnt by it in the past. 
```suggestion
    if (possiblePlugin !=  null) {
```